pwnlib/context/__init__.py

```diff
@@ -308,6 +308,10 @@ class ContextType(object):
         'little'
         >>> context.bits
         32
+
+    .. doctest::
+        :options: +LINUX
```

Comment:
True, this test worked locally because I had the [Ubuntu tool chain](https://gnutoolchains.com/) installed. I want to look into adding them to CI. Maybe mark those with some explicit TODO to see which tests still need to be looked at?

Reply:
`TODO` is there now. Maybe we can get the toolchain installed on all platforms we test in CI and remove this.